### PR TITLE
Fix book path

### PR DESCRIPTION
**Pull request by k1LoW**
Fix #164

### CODE REVIEW

**Comment by k1LoW** [comment]
Path of runbook is needed in `loadBook()` since #156

**Comment by k1LoW** [comment]
Intended error

https://github.com/k1LoW/runn/actions/runs/3127586062/jobs/5074375301#step:6:34

File: testdata/book/http.yml [hunk 1]
```diff
@@ -1,6 +1,8 @@
 desc: Test using HTTP
 runners:
-  req: https://example.com
+  req:
+    endpoint: ${TEST_HTTP_END_POINT:-https:example.com}
+    openapi3: ../openapi3.yml
```
Comment (k1LoW) [inline testdata/book/http.yml:5]: relative path

File: operator_test.go [hunk 2]
```diff
@@ -525,6 +525,29 @@ func TestVars(t *testing.T) {
...
+	for _, tt := range tests {
+		tt := tt
+		t.Run(tt.book, func(t *testing.T) {
+			ts := testutil.HTTPServer(t)
+			t.Setenv("TEST_HTTP_END_POINT", ts.URL)
+			o, err := New(Book(tt.book))
+			if err != nil {
+				t.Fatal(err)
+			}
+			if err := o.Run(ctx); err != nil {
```
Comment (k1LoW) [inline operator_test.go:544]: Run with OpenAPI document

**Review by k2tzumi — APPROVED** [review]
LGTM!
Thank you for your quick response.

**Pull request merged**